README.md

```diff
@@ -1 +1,221 @@
+* Release GitHub Action workflow(s) using the
+  [release-please-action][release-please-action] and the
+  [release-please][release-please] deploy strategy (*optional*)
+  * For wasm libraries, this includes publishing to NPM via
+    [wasm-pack][wasm-pack], reliant on the Cargo version for the wasm package.
+* Test, lint, audit, and code coverage (via [Codecov][codecov]) GitHub Action
+  workflows (*optional*)
+* [Pre-commit][pre-commit] and [rustfmt][rustfmt] opinionated defaults
+* [Dependabot][dependabot] support (*optional*)
+* [Nix flake][nix-flake] support (*optional*)
+* A choice of an [Apache][apache], [MIT][mit], or a dual Apache/MIT license
+
+## Outline
+
+- [Project Templates](#project-templates)
+- [Getting Started](#getting-started)
+- [Contributing](#contributing)
+- [Template References](#template-references)
+- [License](#license)
+
+## Project Templates
+
+This repository contains two sub-templates:
+
+* `rust`: for generating a rust-only library or binary/executable project
+* `rust+wasm` for generating a [cargo workspace][cargo-workspace]
+  with a rust-only crate of the project (library or binary) and another crate
+  for wasm-bindings (library-only), meant for execution in [Node.js][node-js]
+  or running in modern browsers and/or with bundlers like [webpack][webpack].
+
+## Getting Started
+
+### Generating a Rust-Only Project
+
+The `rust` template is designed for generating a rust binary or application
+library.
+
+- Generate a binary project:
+
+  ``` console
+  cargo generate --bin --git https://github.com/fission/rust-template
+  ```
+
+- Generate an application library project:
+
+  ``` console
+  cargo generate --lib --git https://github.com/fission/rust-template
+  ```
+
+- Generate a project from src, locally:
+
+  ``` console
+  cargo generate --lib --path fission/rust-template/
+  ```
+
+*Note on SSH-Keys*: When genearting a project/repository, please be aware
+that RSA keys used with SHA-1 signatures are [no longer supported by
+GitHub][github-rsa]. There is currently [an issue][cargo-generate-issue] in the
+`cargo-generate` repository involving an `id_rsa` default. If you run into an
+associated error using the template, please specify your private key when
+generating a project/repository like so:
+
+```console
+cargo generate -i ~/.ssh/id_ed25519 https://github.com/fission/rust-template
+```
+
+#### 🔋 Batteries Included
+
+- [`tracing`][tracing] for instrumenting Rust programs to collect structured,
+  event-based diagnostic information, going beyond just logging-style
+  diagnostics
+- [`tracing-subscriber`][tracing-subscriber] for Rust binary applications
+  to collect trace data, such as by logging it to standard output, and
+  consume messages emitted by log-instrumented libraries and modules.
+
+### Generating a Rust+Wasm Workspace Project
+
+The `rust+wasm` template is designed for generating a workspace containing both
+a rust native library, as well as one compiled for wasm and leveraging
```

Comment:
```suggestion
a rust native library, as well as one compiled to Wasm and leveraging
```

This whole "Wasm" not "wasm" thing is a bit nitpicky, so feel free to ignore. The use of "wasm" has been fairly consistent here, and that matters more.